### PR TITLE
Pu 1844 implement redis as data store

### PR DESCRIPTION
**Pull request by davidstewarttombola**

### CODE REVIEW

File: config/config.json [hunk 1]
```diff
@@ -1,36 +1,18 @@
 {
     "RequiresAuth": false,
+    "dataSource": "redis",
+    "redisHost": "127.0.0.1",
+    "redisPort": "6379",
```
Comment (tombuildsstuff) [inline config/config.json]: IMO for new data sources it'd be nice to have them in their own section - i.e.:
```
dataSources: {
  redis: {
    host: '',
    port: ''
  }
}
```

Comment (davidstewarttombola) [inline config/config.json]: Sorry mate, opened that PR against the wrong branch, just ignore it :)

Comment (tombuildsstuff) [inline config/config.json]: you can change the branch up the top btw ^ :)

Comment (davidstewarttombola) [inline config/config.json]: yeah I'll not make that mistake again lol

**Pull request closed without merging**